### DISCUSSION TITLE
Is Dataserver opensource?

### DISCUSSION THREAD

**Discussion by immohanravi** [nitrite/nitrite-java#810] [General]
Hi,

First of all this is such a great software, great work actually thank u for providing such a great software.

My question is why Server software is not opensource. I am creating completly open source android app. So just asking

**Comment by anidotnet** [comment]
If you are using 3.x, the datagate server is here - https://github.com/nitrite/nitrite-java/tree/3.x/nitrite-datagate

For current 4.x, the server is being re-written in golang and it is under development. That's why it is not public yet. Once the development is complete, I'll make it open source.
